src/app/organization/employees/employees.component.html

```diff
@@ -0,0 +1,61 @@
+<div class="container m-b-20" fxLayout="row" fxLayoutAlign="end" fxLayoutGap="20px">
+
+  <button mat-raised-button color="primary">
+    <fa-icon icon="upload"></fa-icon>&nbsp;&nbsp;
+    Import Employees
+  </button>
+
+  <button mat-raised-button color="primary">
+    <fa-icon icon="plus"></fa-icon>&nbsp;&nbsp;
+    Create Employees
+  </button>
+
+</div>
+
+<div class="container">
+
+  <div fxLayout="row" fxLayoutGap="20px">
+    <mat-form-field fxFlex>
+      <mat-label>Filter</mat-label>
+      <input matInput (keyup)="applyFilter($event.target.value)">
+    </mat-form-field>
+  </div>
+
+  <div class="mat-elevation-z8">
+
+    <table mat-table [dataSource]="dataSource" matSort>
+
+      <ng-container matColumnDef="displayName">
+        <th mat-header-cell *matHeaderCellDef mat-sort-header> Name </th>
+        <td mat-cell *matCellDef="let employee"> {{ employee.displayName }} </td>
+      </ng-container>
+
+      <ng-container matColumnDef="isLoanOfficer">
+        <th mat-header-cell *matHeaderCellDef mat-sort-header> Is Loan Officer </th>
+        <td mat-cell *matCellDef="let employee"> {{ employee.isLoanOfficer }} </td>
+      </ng-container>
+
+      <ng-container matColumnDef="officeName">
+        <th mat-header-cell *matHeaderCellDef mat-sort-header> Office </th>
+        <td mat-cell *matCellDef="let employees"> {{ employees.officeName }} </td>
+      </ng-container>
+
+      <ng-container matColumnDef="isActive">
+        <th mat-header-cell *matHeaderCellDef mat-sort-header> Status </th>
+        <td mat-cell *matCellDef="let employees">
+          <div [className]="employees.isActive === true ? 'active' : 'inactive'">
+            <fa-icon matTooltip="{{ employees.isActive === true ? 'Active' : 'Inactive' }}" matTooltipPosition="right" icon="stop" size="lg"></fa-icon>
```

Comment:
Let's update this (icon, colors whatever necessary) as per the components currently implemented in the web app: for example, System => Hooks, etc. to ensure consistency. We are using circular icons here.